apps/lib/names.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include <string.h>
+#include <strings.h>             /* strcasecmp() */
```

Comment:
Similarly here and in other files where strings.h is added or `#define _XOPEN_SOURCE 500` is added.